### DISCUSSION TITLE
Customize Url?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#11804] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

how can i change url to 
  https:://mysite.test/admin/team-qs → https:://mysite.test/admin/teamQ
  @leandrocfe do you know ?

**Comment by leandrocfe** [comment, marked as answer]
slug in your page?
https://filamentphp.com/docs/3.x/panels/pages#customizing-the-page-url

**Reply by zeeshantariq08** [reply]
Great! it worked , thank you
